common/api/http.go

```diff
@@ -37,15 +37,15 @@ func (s HttpScheme) String() string {
 	return string(s)
 }
 
-type ApiHttp struct {
+type APIHttp struct {
```

Comment:
type name will be used as api.APIHttp by other packages, and that stutters; consider calling this Http